config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+   config.default_per_page = 15
```

Comment:
This will affect the whole app. Kaminari's default is 25 per page and I would rather stick with that.